paramiko_expect.py

```diff
@@ -56,12 +56,14 @@ class SSHClientInteraction(object):
                     real-time as it is being performed (especially useful
                     when debugging)
     :param encoding: The character encoding to use.
+    :param lines_to_check: The number of last few lines of the output to
+                           look at, while matching regular expression(s)
     """
 
     def __init__(
         self, client, timeout=60, newline='\r', buffer_size=1024,
         display=False, encoding='utf-8', output_callback=default_output_func,
-        tty_width=80, tty_height=24
+        tty_width=80, tty_height=24, lines_to_check=2
```

Comment:
This should probably default to `1` to avoid breaking backwards compatibility.

Reply:
Done. Thanks.